tests/Feature/PageArchiveTest.php

```diff
@@ -43,4 +43,43 @@ public function it_orders_streams_from_latest_to_oldest(): void
                 'Finished three days ago',
             ]);
     }
+
+    /** @test */
+    public function we_can_search_for_streams_on_title(): void
+    {
+        // Arrange
+        Stream::factory()->finished()->create(['title' => 'Finished one day ago']);
```

Comment:
Please use different titles. The first impression is that it has something today with the finished time of a stream. Using simple names like `Stream #1` (etc.) will make it more clear that is just about the name.